### ISSUE TITLE
Conflict with Pods and the Sage theme wrapper

### ISSUE THREAD

**Issue by jackhogg** [pods-framework/pods#5350]
Pod seems to be preventing template_include from firing, when a custom page template is used (e.g., 'X-page.blade.php',) and the page then merely displays the blade template code as it's written in the file. Is there a way to resolve or work around this? Sage's discussion board and GitHub issues mention this being an issue, but suggest that the problem is with how Pods handles custom templates.

**Comment by jimtrue** [comment]
@jackhogg Can you please provide us links to issues from the Sage theme wrapper GitHub? We don't actually do anything special with custom templates, our Auto Template functionality feeds into `the_content` filter but without more details to the issue it's very hard for us to work with the other provider to troubleshoot.

**Comment by jimtrue** [comment]
@jackhogg Are you referring to this issue which is the only time I found Pods mentioned in 'sage' github, back from 2014: https://github.com/roots/sage/issues/1129

I found two places where we are calling `template_redirect` from that issue:
https://github.com/pods-framework/pods/blob/edc95828c1fc94f8003d0fc8d2f3ac7bde69493b/components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php#L22

and 

https://github.com/pods-framework/pods/blob/d80d8d009850b6d18ff51566f85af683c891690b/components/Pages.php#L889

Dropping this for @sc0ttkclark to have a look at.

**Comment by sc0ttkclark** [comment]
The only place that would be incompatible with Sage here is the way we handle template_redirect for pod pages (the second piece of code Jim sent). Is this specifically in regards to your use of Pod Pages and choosing a custom page template file?

**Comment by jackhogg** [comment]
@sc0ttkclark Yes, it seem to be specifically related to Pods Pages, when using a custom page template.

**Comment by sc0ttkclark** [comment]
In that case, this isn’t easily fixed within Pod Pages. It requires more rewriting of that code and testing.

**Comment by sc0ttkclark** [comment]
I've got some fixes for this but it [was not easy](https://github.com/roots/sage/issues/1129#issuecomment-1084019265) to figure out a workaround.

Commits in Pods 2.8.11 that help bring the first initial compatibility work to support Sage can be found here:

* 7e3ccfd125ea4b38c904206609b7f6f6c22adb9e
* d8eea8716b953f8d052e90f0d04838f1c594cdc8
